flashinfer_bench/bench/timing/__init__.py

```
@@ -0,0 +1,58 @@
+from __future__ import annotations
+
+import statistics
+from typing import Any, Callable, List
+
+import torch
+from flashinfer.testing import bench_gpu_time_with_cupti
+
+from ._common import _device_lock
+from .split_timing import SplitTimingMetrics, time_runnable_split_timing
+
+__all__ = ["time_runnable", "time_runnable_split_timing", "SplitTimingMetrics"]
```

Ruff (0.15.21)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)